src/static/css/nav-primary.less

```diff
@@ -66,398 +66,320 @@
+        // The .no-js selector enables simple hover menu support for users w/o JS.
+        &[aria-expanded="true"],
+        .no-js .primary-nav_item:hover & {
+            max-height: unit(400px / @base-font-size-px, em);
 
-    // Transition on.
-    .has-transition .sub-nav_wrapper[aria-expanded="true"] {
-        transition: 0.275s max-height ease-out;
-    }
+            border-top: 1px solid @gray-50;
+            border-bottom: 4px solid @gray-50;
+        }
 
-    .sub-nav_inner-wrapper {
-        padding: @grid_gutter-width 0;
-    }
+        &_title {
+            padding-bottom: @grid_gutter-width / 2;
 
-    .sub-nav_menu {
-        .grid_column(8);
-    }
-
-    .sub-nav_menu-list {
-        .grid_column(1, 3);
-        vertical-align: top;
-        &:first-of-type {
-            border-left-width: 0;
+            font-size: 1em;
         }
-    }
 
-    .sub-nav_menu-list .list_link {
-       .u-inline-block();
-        padding: unit(5px / @font-size-large, em)
-                 unit(15px / @font-size-large, em);
-        border-left-style: solid;
-        border-left-width: 5px;
-        font-size: unit(@font-size-large / @base-font-size-px, em);
-        .u-link__colors(@pacific, @pacific, @black, @black, @black,
-                        transparent, transparent, @green, @green, @green);
-        &__current {
-            .u-link__colors(@black, @black, @black, @black, @black,
-                            @green, @green, @green, @green, @green);
-            cursor: default;
+        &_link {
+            .u-link__colors(@pacific, @black);
         }
 
-        .webfont-regular();
-    }
+        .list_link {
+            .u-link__colors(@pacific, @pacific, @black, @black, @black,
+                            transparent, transparent, @green, @green, @green);
 
-    .sub-nav_menu-list .list_item {
-        display: block;
-        white-space: nowrap;
-        line-height: 2;
-    }
+            padding-top: unit(10px / @font-size-large, em);
+            padding-bottom: unit(10px / @font-size-large, em);
+            padding-left: unit(15px / @font-size-large, em);
 
-    .sub-nav_media {
-        position: relative;
-        padding: 0 @grid_gutter-width;
-        margin-top: 0;
-        .grid_column(4);
-    }
 
+            border-left-style: solid;
+            border-left-width: 5px;
+            font-size: unit(@font-size-large / @base-font-size-px, em);
 
+            &__current {
+                .u-link__colors(@black, @black, @black, @black, @black,
+                                @green, @green, @green, @green, @green);
 
-});
-
-.respond-to-max(999px, {
-    .sub-nav_menu {
-        width: 100%;
+                cursor: default;
+            }
+        }
     }
+});
 
+.respond-to-max(1000px, {
     .sub-nav_media {
         display: none;
     }
 });
 
-.respond-to-range(1000px, @wall-min, {
-    .sub-nav_media {
-        padding: 0 @grid_gutter-width/2;
+.respond-to-min(1001px, {
```

Comment:
After discussing with Steven, we came to the conclusion that it's probably best to avoid variables for one-off breakpoints to avoid breaking from the default breakpoints unless it's necessary (as it is in this case).